frontend/src/containers/LoginPage/LoginPage.js

```diff
@@ -3,41 +3,41 @@ import SignUp from '../../components/SignUp/SignUp.js';
 import SignIn from '../../components/SignIn/SignIn.js';
 import Footer from '../../components/Footer/Footer.js'
 import LogInPageHeader from '../../components/LogInPageHeader/LogInPageHeader.js';
+import ThankYouCard from '../../components/ThankYouCard/ThankYouCard.js';
 import './LoginPage.css';
-//import axios from 'axios';
 
 class LoginPage extends Component {
+
   constructor(props) {
     super(props)
     this.state = {
-        email: "",
-        encrypted_password: "",
-        auth_token: "",
-        isAuthorized: false
+      isAuthorized: false
     }
-    this.handleInputChange = this.handleInputChange.bind(this);
-    //this.login = this.login.bind(this);
   }
 
-  handleInputChange(event) {
-      this.setState({ [event.target.name]: event.target.value });
+  mimicLogin = (event) => {
+    event.preventDefault();
+    this.setState(prevState => ({
+      isAuthorized: !prevState.isAuthorized
+    }));
   }
-
   
- /* login(event) {
-    axios.post('api/v1/login', {
-        email: this.state.email,
-        encrypted_password: this.state.encrypted_password
-      }).then(response => {
-      this.setState({ 
-        isAuthorized: true,
-        email: response.data.email
-
-      })
-    })
-    .catch(console.error)
-  }
-  */
+ renderLoginOrThankYouCard = () => {
+   if (this.state.isAuthorized) {
+     return (
+      <div className="column mx-auto thankYouColumn"> 
+      <ThankYouCard/>
+    </div>
+     )
+   } else {
+     return (
+      <div className="row">
+        <div className="col loginColumnOne"><SignUp mimicLogin={(event) => this.mimicLogin(event)} /></div>
+        <div className="col loginColumnTwo"><SignIn mimicLogin={(event) => this.mimicLogin(event)} /></div>
```

Comment:
I think `mimicLogin=this.mimicLogin` should work.

Reply:
Since we didn't end up using this fake functionality, I actually made all these changes in the `feature/156/sign-up-for-account` branch. I'm not going to worry about changing this, as I don't think we'll need to merge this in for the next sprint. We'll have full log in functionality! What do you think?